lib/pacer_test.go

```diff
@@ -0,0 +1,148 @@
+package vegeta
+
+import (
+	"math"
+	"testing"
+	"time"
+)
+
+func TestRatePacing(t *testing.T) {
```

Comment:
What about all the edge cases ?

- zero hits per second
- 1 hit per zero seconds
- zero hits per zero seconds
- other cases when the second argument turns out to become `true`

Reply:
I just copied these tests over from the other file, I agree they need some additions. I'm happy to augment them with more tests.

Reply:
> I'm happy to augment them with more tests.

Covering the edge cases seems important.